src/commonTest/kotlin/kotools/types/number/StrictlyNegativeIntTest.kt

```diff
@@ -71,6 +72,39 @@ class StrictlyNegativeIntTest {
             .shouldHaveAMessage()
     }
 
+    @ExperimentalNumberApi
+    @Test
+    fun number_toStrictlyNegativeIntOrNull_should_not_be_null_with_a_strictly_negative_Int() {
+        val number: Number = StrictlyNegativeInt.random().toInt()
+        val result: StrictlyNegativeInt? = number.toStrictlyNegativeIntOrNull()
+        result?.toInt() shouldEqual number
```

Comment:
We should check first that the `result` shouldn't be `null` by using the [`shouldBeNotNull`](https://github.com/kotools/types/blob/39c92ca9cd9b0373f5613b7e5da6950e278715ba/src/commonTest/kotlin/kotools/types/Assertions.kt#L20) function.

```suggestion
        result.shouldBeNotNull().toInt() shouldEqual number
```